cmd/scollector/main.go

```diff
@@ -50,14 +50,18 @@ var (
 )
 
 type scollectorHTTPTransport struct {
-	UserAgent string
+	UserAgent    string
+	XAccessToken string
 	http.RoundTripper
 }
 
 func (t *scollectorHTTPTransport) RoundTrip(req *http.Request) (*http.Response, error) {
 	if req.Header.Get("User-Agent") == "" {
 		req.Header.Add("User-Agent", t.UserAgent)
 	}
+	if t.XAccessToken != "" {
+		req.Header.Add("X-Access-Token", t.XAccessToken)
```

Comment:
We already have an AuthToken in the collect package. Should be as easy as setting that from the scollector config.

https://github.com/bosun-monitor/bosun/blob/master/collect/collect.go#L63